### ISSUE TITLE
should `float_mult` have a doc page?

### ISSUE THREAD

**Issue by dsclassen** [bl831/bin_stuff#8]
`float_add`, and `float_func` have dedicated doc pages. Should `float_mult` have a page too?

**Comment by jmholton** [comment]
no. In fact float_mult is basically obsolesced by float_func.  Could delete?
